src/popup/router/pages/NamingSystem.vue

```diff
@@ -51,23 +48,15 @@
         <ae-filter-list v-if="!moreAuInfo.visible">
           <p style="margin:0">{{ $t('pages.namingSystemPage.filtersBy') }}</p>
           <div class="filters">
-            <ae-filter-item class="au-filter notround" @click.native="filterByMine" :active="byMine ? true : false">
-              {{ $t('pages.namingSystemPage.filterByMine') }}
-            </ae-filter-item>
-            <ae-filter-item class="au-filter notround" @click.native="filterBySoonest" :active="bySoonest ? true : false">
-              {{ $t('pages.namingSystemPage.filterBySoonest') }}
-            </ae-filter-item>
-            <ae-filter-item class="au-filter notround" @click.native="filterByCharLength" :active="byCharLength ? true : false">
-              {{ $t('pages.namingSystemPage.filterByCharLength') }}
-            </ae-filter-item>
-            <ae-filter-item class="au-filter notround" @click.native="filterByBid" :active="byBid ? true : false">
-              {{ $t('pages.namingSystemPage.filterByBid') }}
-            </ae-filter-item>
+            <ae-filter-item class="au-filter notround" @click.native="filterType = 'mine'" :active="filterType == 'mine' ? true : false"> {{ $t('pages.namingSystemPage.filterByMine') }} </ae-filter-item>
```

Comment:
We can leave the expression just with `:active="filterType === 'mine'`, because it returns boolean